### DISCUSSION TITLE
RPC Method 'wallet_switchEthereumChain'

### DISCUSSION THREAD

**Discussion by KieranAltman** [consenlabs/webview#21] [Q&A] [answered]
Will you support the RPC method of 'wallet_switchEthereumChain'?

**Comment by unix** [comment, marked as answer]
Currently, we don't support `wallet_switchEthereumChain`, you can use the `showAccountSwitch` in the [SDK](https://imtoken.gitbook.io/developers/products/webview/sdk#showaccountswitch) for now instead.

This is a practical suggestion and I have submitted this feature to the internal team for review, but it will take some time to discuss and implement(or not). I will come back to update this issue when there are future updates.

**Reply by KieranAltman** [reply]
我想尽可能的兼容手机钱包，所以先使用 `wallet_addEthereumChain` 处理了
